uber/models.py

```diff
@@ -1144,16 +1144,14 @@ def badge_cost(self):
 
     @property
     def age_group_conf(self):
```

Comment:
We perform validations against this property, but we don't update ``age_group`` until we're already passed validations.  Therefore, when both the ``age_group`` and the ``birthdate`` are set, we want to use the ``birthdate`` to determine which of the ``AGE_GROUP_CONFIGS`` we select.  Otherwise we'd run into a situation such as the following
- attendee exists who is wrongly marked as under 18 and who is not signed up for volunteering
- admin corrects their birthdate and marks them as volunteering
- validation checks ``age_group`` which has not yet updated, and says they cannot volunteer

This fixes that problem.